### PR TITLE
Reworks eth send raw transaction flow with qtum utxos

### PR DESCRIPTION
**Pull request by IntegralTeam**
Revisions Ethereum send raw transaction method:
  * pkg/eth/rpc_types.go: add SendRawTransactionRequest and SendRawTransactionResponse types;
  * pkg/qtum/rpc_types.go: add SendRawTransactionRequest and SendRawTransactionResposne types;
  * pkg/qtum/method.go: add Method.SendRawTransaction();
  * pkg/transformer/eth_sendRawTransaction.go: revision ProxyETHSendRawTransaction.Request().Implements get qtum UTXOs RPC proxy:
  * pkg/eth/rpc_types.go: add GetUTXOsRequest{};
  * pkg/transformer/util.go: add convertETHAddress() and convertQtumAddress();
  * pkg/qtum/rpc_types.go: actualize ListUnspentRequest{}, ListUnspentResponse{}; add ListUnspentQueryOptions{};
  * pkg/transformer/qtum_getUTXOs.go: add ProxyQTUMGetUTXOs{};
  * pkg/transformer/transformer.go: add ProxyQTUMGetUTXOs{} to DefaultProxies().

### CODE REVIEW

File: pkg/eth/rpc_types.go [hunk 1]
```diff
@@ -67,6 +70,15 @@ func (t *SendTransactionRequest) GasPriceHex() string {
...
 }
 
+// ========== eth_sendRawTransaction ============= //
+
+type (
+	// Presents hexed string of a raw transaction
+	SendRawTransactionRequest [1]string
+	// Presents hexed string of a transaction hash
+	SendRawTransactionResponse [1]string
+)
```
Comment (VoR0220) [inline pkg/eth/rpc_types.go:80]: Unnecessary as sendRawTransaction has already been implemented. See eth_sendRawTransaciton under the transformer package.

File: pkg/qtum/method.go [hunk 2]
```diff
@@ -206,3 +206,10 @@ func (m *Method) ListUnspent(req *ListUnspentRequest) (resp *ListUnspentResponse
 	}
 	return
 }
+
+func (m *Method) SendRawTransaction(req *SendRawTransactionRequest) (resp *SendRawTransactionResposne, err error) {
```
Comment (VoR0220) [inline pkg/qtum/method.go:210]: See other comment. This is unneccesary as already implemented in transformer package.

File: pkg/qtum/rpc_types.go [hunk 3]
```diff
@@ -1216,6 +1217,15 @@ type (
...
 )
 
+// ======== sendrawtransaction ========= //
+
+type (
+	// Presents hexed string of a raw transcation
+	SendRawTransactionRequest [1]string
+	// Presents hexed string of a transaction hash
+	SendRawTransactionResposne [1]string
+)
```
Comment (VoR0220) [inline pkg/qtum/rpc_types.go:1227]: see above comments.

File: pkg/transformer/util_test.go [hunk 4]
```diff
@@ -41,3 +44,35 @@ func TestQtumAmountToEthValue(t *testing.T) {
...
 }
+
+func TestAddressesConvertion(t *testing.T) {
+	t.Parallel()
+
+	inputs := []struct {
+		qtumChain   string
+		ethAddress  string
+		qtumAddress string
+	}{
```
Comment (VoR0220) [inline pkg/transformer/util_test.go:55]: Lets get a couple more tests in here. see these notes for some good test cases that I would want to ensure converted. https://github.com/hayeah/openzeppelin-solidity/blob/qtum/QTUM-NOTES.md

Comment (IntegralTeam) [reply pkg/transformer/util_test.go:55]: - The issue is solved. Please, see commit hash ac69fa591332c2c93018bc9c55c126346c931ab5

**Pull request merged**